tree-sitter-stack-graphs/tests/it/test.rs

```diff
@@ -94,9 +95,18 @@ fn check_test(
         expected_successes + expected_failures,
         assertion_count,
     );
+
     let mut globals = Variables::new();
     for fragments in &test.fragments {
         globals.clear();
+
+        globals
+            .add(
+                FILE_PATH_VAR.into(),
+                fragments.path.to_str().unwrap().into(),
+            )
+            .expect("failed to add file path variable");
```

Comment:
The CI failure is probably because this is still before loading the globals from the test, which can set file path too.

Reply:
Maybe adding a `add_if_not_set` (not sure about the naming) convenience function to Variables would be a nice addition. Let me fix this